### DISCUSSION TITLE
Optional Resource Type in related identifiers

### DISCUSSION THREAD

**Discussion by rkrug** [eblondel/zen4R#89] [Q&A] [answered]
Is it possible to set the optional Resource type of the related identifier from zen4R?

Thanks.

**Comment by eblondel** [comment, marked as answer]
I've created an issue at #90 to support this enhancement as it is not yet added.

**Reply by rkrug** [reply]
Thanks - any idea when it will be implemented? I am preparing a a few hundred deposits and it would be great if I could use it for that. I have some buffer time to deposit them.

**Reply by eblondel** [reply]
i'll do my best to write the code by the end of the week, as it should be easy

**Reply by eblondel** [reply]
see also #91

**Reply by rkrug** [reply]
This is more than perfect. Thanks.

**Reply by eblondel** [reply]
ok reinstall from github/master i've pushed some commits to #90 
now you can specify the `resource_type` as third argument (optional) of `addRelatedIdentifier`

**Comment by rkrug** [comment]
Thanks a lot- you re a star.
